### PR TITLE
[staging-17.0] Merge master to staging-17.0 up to 2f4684ae9

### PR DESCRIPTION
**Pull request by darcywong00**
12 of x Merging master to staging-17.0 up to 2f4684ae9

How does `und-fonipa` from kreative_superipa.keyboard_info get migrated?

### CODE REVIEW

**Comment by mcdurdin** [comment]
> How does `und-fonipa` from kreative_superipa.keyboard_info get migrated?

https://github.com/keymanapp/keyman/issues/11046

File: release/fv/fv_plains_cree/source/fv_plains_cree.kps [hunk 1]
```diff
@@ -43,6 +43,54 @@ region of Canada. This is part of a set of keyboards from FirstVoices.</Descript
...
+      <Description>File fv_plains_cree.kps</Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.kps</FileType>
+    </File>
+    <File>
+      <Name>fv_plains_cree.kvks</Name>
+      <Description>File fv_plains_cree.kvks</Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.kvks</FileType>
+    </File>
```
Comment (mcdurdin) [inline release/fv/fv_plains_cree/source/fv_plains_cree.kps:57]: I wonder if these are fixed in a later PR? They shouldn't really be here.

**Review by mcdurdin — APPROVED** [review]

File: release/fv/fv_plains_cree/source/fv_plains_cree.kps (hunk 1 above)
Comment (darcywong00) [reply release/fv/fv_plains_cree/source/fv_plains_cree.kps:57]: Welp - they're still on master
https://github.com/keymanapp/keyboards/blob/8b7824d21375a64fe32b46afa349651479d83153/release/fv/fv_plains_cree/source/fv_plains_cree.kps#L43-L54

Comment (darcywong00) [reply release/fv/fv_plains_cree/source/fv_plains_cree.kps:57]: TODO for #2781

**Pull request merged**